src/components/ErrorPage/ErrorPage.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import { useStaticQuery, graphql } from 'gatsby';
+import Img from 'gatsby-image';
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
+import { Link, Typography } from '@material-ui/core';
+
+import pixelheart from './pixelheart.png';
+import hackheart from './hackheart.png';
+const styles = theme => ({
+	container: {
+		display: 'flex',
+		alignItems: 'center',
+		justifyContent: 'center',
+		flexDirection: 'column',
+		marginTop: '5%'
+	},
+	content: {
+		fontFamily: theme.typography.body1.fontFamily,
+		fontSize: theme.typography.fontSize * 1.3,
+		lineHeight: 1.76,
+		marginTop: theme.spacing(7),
+		marginBottom: theme.spacing(4),
+		marginLeft: theme.spacing(2.75),
+		marginRight: theme.spacing(2.75),
+		[theme.breakpoints.down('xs')]: {
+			fontSize: theme.typography.fontSize,
+			lineHeight: 1.57,
+			marginTop: theme.spacing(3),
+			marginBottom: theme.spacing(3)
+		}
+	},
+	pageContainer: {
+		minHeight: '100vh'
+	}
+
+});
+
+
+function ErrorPage({ classes }) {
+	const data = useStaticQuery(graphql`
+      query {
+        file(relativePath: {eq: "404/bowimg-removebg.png"}) {
+            childImageSharp {
+                fixed(height: 400) {
+                  base64
+                  width
+                  height
+                  src
+                  srcSet
+                }
+            }
+          }
+      }
+    `);
+	return (
+		<div className={classes.pageContainer}>
+			<div className={classes.container}>
+				<div><Typography variant='h3' >NOT FOUND</Typography></div>
+				<p>
+					<Typography variant='h5' >
+                        Oops! This page has either does not exist or has been taken down.
+					</Typography>
+				</p>
+				<div><Img fixed={data.file.childImageSharp.fixed}></Img></div>
+			</div>
+			<div className={classes.container}>
+				<Typography display='inline' variant='h4'>With love <img src={pixelheart}
+					onClick={hidepic}
+					alt='404 Meme' width="" height="30"></img> from Hack </Typography>
```

Comment:
The indentation for `img` tag. Let's make it better. Use consistent quotes as well. And why `width=""`? Why not 0 or not setting it?

```jsx
<img 
  src={pixelheart}
  onClick={hidepic}
  alt="404 Meme"
  // ...
/>
```